test/src/UnitTests.hpp

```diff
@@ -0,0 +1,58 @@
+// Header for SFML unit tests.
+//
+// For a new test case, include this header and not <catch.hpp> directly.
+// This ensures that string conversions are visible and can be used by Catch for debug output.
+
+#ifndef SFML_UNITTESTS_HPP
+#define SFML_UNITTESTS_HPP
+
+#include <SFML/System/Vector2.hpp>
+#include <SFML/System/Vector3.hpp>
+#include <SFML/Graphics/Rect.hpp>
+#include <sstream>
+#include <catch.hpp>
```

Comment:
I'd suggest to move <catch.hpp> higher (maybe so that it's the first included header?), because the list of SFML headers included here may become bigger in the future.

Reply:
I moved it up.